infra-templates/k8s-support/v2.0.0-alpha1/compose.yml

```diff
@@ -17,6 +18,20 @@ services:
     image: {{$kubectlImage}}
     privileged: true
 
+  rancher-kubernetes-agent:
+    labels:
+      io.rancher.container.create_agent: "true"
+      io.rancher.container.agent.role: agent
+      io.rancher.container.agent_service.labels_provider: "true"
```

Comment:
@galal-hussein we do not need `io.rancher.container.agent_service.labels_provider` anymore